corehq/apps/hqwebapp/static/hqwebapp/scss/commcarehq/_forms.scss

```diff
@@ -52,6 +53,10 @@ legend {
   padding: $spacer 0;
   margin: 0 0 $spacer 0;
 
+  .field-control:first-child {
+    padding-left: ($grid-gutter-width - $spacer) / 2;
```

Comment:
Curious why this was added here rather than `.row > .field-control:first-child { ... }` rule with `flex-shrink: initial;`? Or could `flex-shrink: initial;` be moved in here too rather than putting it with the extension classes?

Reply:
because the fix is only needed for the `form-actions` children

Reply:
Got it. Hadn't noticed that this was only applied to `form-actions`